src/lib/dnssd/Discovery_ImplPlatform.h

```diff
@@ -49,7 +49,7 @@ class DiscoveryImplPlatform : public ServiceAdvertiser, public Resolver
 
     // Members that implement Resolver interface.
     void SetResolverDelegate(ResolverDelegate * delegate) override { mResolverDelegate = delegate; }
-    CHIP_ERROR ResolveNodeId(const PeerId & peerId, Inet::IPAddressType type) override;
+    CHIP_ERROR ResolveNodeId(const PeerId & peerId, Inet::IPAddressType type, Resolver::CacheBypass dnssdCacheBypass) override;
```

Comment:
Would it be possible to set it to `Dnssd::Resolver::CacheBypass::Off` by default ? I believe in general the cache should not be bypassed.

Reply:
Yes, that's a good idea, done.